game/graphics/objvisual.cpp

```diff
@@ -114,30 +114,30 @@ void ObjVisual::setType(Type t) {
   type = t;
   }
 
-void ObjVisual::setVisual(const Daedalus::GEngineClasses::C_Item& hitem, World& world, bool staticDraw) {
+void ObjVisual::setVisual(const phoenix::c_item& hitem, World& world, bool staticDraw) {
   cleanup();
 
-  if(FileExt::hasExt(hitem.visual.c_str(),"ZEN")) {
+  if(FileExt::hasExt(hitem.visual,"ZEN")) {
     setType(M_Bundle);
-    bundle = VobBundle(world,hitem.visual.c_str(),(staticDraw ? Vob::Static : Vob::None));
+    bundle = VobBundle(world,hitem.visual,(staticDraw ? Vob::Static : Vob::None));
     } else {
     setType(M_Mesh);
     mesh.view = world.addView(hitem);
     }
   }
 
-void ObjVisual::setVisual(const ZenLoad::zCVobData& vob, World& world, bool staticDraw) {
+void ObjVisual::setVisual(const phoenix::vob& vob, World& world, bool staticDraw) {
   cleanup();
 
-  const bool enableCollision = (vob.cdDyn || vob.cdStatic);
+  const bool enableCollision = (vob.cd_dynamic || vob.cd_static);
 
   // *.ZEN; *.PFX; *.TGA; *.3DS; *.MDS; *.ASC; *.MMS
-  if(FileExt::hasExt(vob.visual,"ZEN")) {
+  if(FileExt::hasExt(vob.visual_name,"ZEN")) {
     setType(M_Bundle);
-    bundle = VobBundle(world,vob.visual,(staticDraw ? Vob::Static : Vob::None));
+    bundle = VobBundle(world,vob.visual_name,(staticDraw ? Vob::Static : Vob::None));
     }
-  else if(FileExt::hasExt(vob.visual,"PFX") || FileExt::hasExt(vob.visual,"TGA")) {
-    if(vob.visualCamAlign==0 && FileExt::hasExt(vob.visual,"TGA")) {
+  else if(FileExt::hasExt(vob.visual_name,"PFX") || FileExt::hasExt(vob.visual_name,"TGA")) {
+    if(vob.camera_alignment==phoenix::camera_lock_mode::none && FileExt::hasExt(vob.visual_name,"TGA")) {
```

Comment:
`camera_lock_mode` is misleading name, can be `sprite_to_camera_alignment` or similar